src/include/mempool.h

```diff
@@ -356,9 +356,12 @@ class pool_allocator {
     shard_t *shard = pool->pick_a_shard();
     shard->bytes += total;
     shard->items += n;
+    // FIXME: type_t::items isn't sharded
+#if !defined(WITH_SEASTAR) || defined(WITH_ALIEN)
     if (type) {
       type->items += n;
     }
+#endif
```

Comment:
since type is not sharded, so still has the problem all shards update the same variable in crimson+alien scenario, right?

Reply:
should be addressed by https://github.com/ceph/ceph/pull/53130/commits/9042b77fff476449be0ad010c31fd4d9e4ac0238